extlinks/links/migrations/0010_data_link_event_id_hash.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.14 on 2023-02-15 17:15
+import hashlib
+from django.db import migrations
+
+
+def add_hash_link_event_id(apps, schema_editor):
+    LinkEvent = apps.get_model("links", "LinkEvent")
+    unhashed = LinkEvent.objects.filter(hash_link_event_id__exact='')
+    for i in range(100):
```

Comment:
this could have been more general if I actually used the number of objects as the basis for chunking up this work. Maybe something like `range((unhashed.count() + 100000 // 2) // 100000)` but I was in a rush.